### DISCUSSION TITLE
docker 安装失败 如何解决

### DISCUSSION THREAD

**Discussion by st1341815** [1Panel-dev/1Panel#5386] [Q&A]
curl: (28) Operation timed out after 300233 milliseconds with 0 out of 0 bytes received
sh: get-docker.sh: 没有那个文件或目录
[1Panel Log]: ... 启动 docker 
Failed to execute operation: No such file or directory
Failed to start docker.service: Unit not found.
[1Panel Log]: docker 安装失败

**Comment by wanghe-fit2cloud** [comment]
可以考虑使用安装包：
https://1panel.cn/docs/installation/package_installation/
